influxdb_client_3/__init__.py

```diff
@@ -194,10 +216,15 @@ def query(self, query, language="sql", mode="all", database=None,**kwargs ):
             database = self._database
         
         try:
-            headers = [(b"authorization", f"Bearer {self._token}".encode('utf-8'))]
-    
             # Create an authorization header
-            _options = FlightCallOptions(headers=headers, **kwargs)
+            optargs = {
+                "headers": [(b"authorization", f"Bearer {self._token}".encode('utf-8'))],
+                "timeout": 300
+            }
+            opts = self._merge_options(optargs, kwargs)
+            _options = FlightCallOptions(
+                headers=opts["headers"],
+                timeout=opts["timeout"])
```

Comment:
hmm this still doesn't work as the `kwargs` don't need to be merged with the header in this case if we look at the FlightCallOptions: https://arrow.apache.org/docs/python/generated/pyarrow.flight.FlightCallOptions.html

I belive we should do it like follows:
```
 def query(self, query, language="sql", mode="all", database=None,**kwargs ):
        """
        Query data from InfluxDB.
	@@ -185,7 +207,7 @@ def query(self, query, language="sql", mode="all", database=None,**kwargs ):
        :type mode: str
        :param database: The database to query from. If not provided, uses the database provided during initialization.
        :type database: str
        :param kwargs: FlightClientCallOptions for the query.
        :return: The queried data.
        """

	@@ -194,10 +216,15 @@ def query(self, query, language="sql", mode="all", database=None,**kwargs ):
            database = self._database

        try:
            # Create an authorization header
            optargs = {
                "headers": [(b"authorization", f"Bearer {self._token}".encode('utf-8'))],
                "timeout": 300
            }
            opts = self._merge_options(optargs, kwargs)
            _options = FlightCallOptions(
                headers=opts["headers"],
                timeout=opts["timeout"])
```